src/qml/QFieldSettings.qml

```diff
@@ -249,15 +249,15 @@ Page {
                     Layout.minimumHeight: contentHeight
                     MouseArea {
                         anchors.fill: parent
-                        onClicked: useNativeCameraCheckBox.toggle()
+                        onClicked: nativeCameraCheckBox.toggle()
                     }
                 }
 
                 Label {
                     padding: 8 * dp
                     topPadding: 0
                     leftPadding: 22 * dp
-                    text: qsTr( "Warning: native camera function is unstable on recent Android versions." )
+                    text: qsTr( "If enabled, the user can choose the system camera app to use." )
```

Comment:
```suggestion
                    text: qsTr( "If disabled, QField will use a minimalist internal camera instead of the camera app on the device.<br>Tip: Enable this option and install the open camera app to create geo tagged photos." )
```

Or

```suggestion
                    text: qsTr( "If disabled, QField will use a minimalist internal camera built on :fish: and :banana: instead of the camera app on the device.<br>Tip: Enable this option and install the open camera app to create geo tagged photos." )
```

Reply:
:laughing: